resources/projectPortTesting/projectGo/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+)
+
+func main() {
+
+	//api := mux.NewRouter()
+	http.HandleFunc("/", HelloHandler)
+	//http.Handle("/hello", api)
+
+	fmt.Println("Listening on localhost:8080")
+	http.ListenAndServe(":8080", nil)
+}
+
+func HelloHandler(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(http.StatusOK)
+	hostname, err := os.Hostname()
+	if err != nil {
+		fmt.Println("unable to get hostname")
+	}
+
+	// fmt.Fprintf(w, "Hello from Go! %s on %s\n", time.Now(), hostname)
```

Comment:
are these comments intentional or just left over from testing?

Reply:
This is a test repo which is imported to be used for port recognition. I can clean it a little bit, but I guess there are a lot of cases like this in the rest of the code. Because I totally agree that this has to be cleaned, I would propose to have a separate issue for test projects grooming/cleaning. WDYT?

Reply:
Agreed, we can clean this test for now

Reply:
Created issue for tests cleaning #184